repository/src/main/resources/db/changelog/db.changelog-rdm-2917.xml

```diff
@@ -0,0 +1,504 @@
+
+
+
+        <!-- START - event_acl table -->
+        <!-- create the new event_acl table -->
+        <createTable tableName="event_acl">
+            <column name="id" type="serial">
+                <constraints nullable="false"/>
+            </column>
+            <column name="event_id" type="integer">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_role_id" type="integer">
+                <constraints nullable="true"/>
+            </column>
+            <column name="create" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="read" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="update" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="delete" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="live_from" type="date" >
+                <constraints nullable="true"/>
+            </column>
+            <column name="live_to" type="date" >
+                <constraints nullable="true"/>
+            </column>
+            <column name="created_at" type="datetime" defaultValue="now()">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addPrimaryKey columnNames="id"
+                       constraintName="pk_event_acl"
+                       tableName="event_acl"/>
+
+        <!-- Add foreign key for event_id column linking to event table -->
+        <addForeignKeyConstraint baseColumnNames="event_id"
+                                 baseTableName="event_acl"
+                                 constraintName="fk_event_acl_event_id"
+                                 referencedColumnNames="id"
+                                 referencedTableName="event"/>
+
+        <!-- Add foreign key for user_role_id column linking to user_role table -->
+        <addForeignKeyConstraint baseColumnNames="user_role_id"
+                                 baseTableName="event_acl"
+                                 constraintName="fk_event_acl_user_role_id_user_role_id"
+                                 referencedColumnNames="id"
+                                 referencedTableName="user_role"/>
+
+        <!-- Add unique constraint on event_id and user_role_id columns for the event_acl table -->
+        <addUniqueConstraint columnNames="event_id, user_role_id"
+                             constraintName="unique_event_acl_event_id_user_role_id"
+                             tableName="event_acl"/>
+
+        <!-- copy table data to the new event_acl table -->
+        <sql>
+            insert into event_acl select * from event_acl;
+        </sql>
+
+        <!-- Add new role_id column to the event_acl table -->
+        <addColumn tableName="event_acl">
```

Comment:
I don't see the benefit of splitting this column creation from the table creation